Source/Core/Core/PowerPC/Jit64/JitAsm.cpp

```diff
@@ -16,6 +16,8 @@
 #include "Core/PowerPC/Jit64Common/Jit64PowerPCState.h"
 #include "Core/PowerPC/PowerPC.h"
 
+#include <limits.h>
```

Comment:
```suggestion
#include <climits>
```

Also our current convention is to place standard includes after the corresponding header include for the cpp file (aka, the first one).